action/src/handlePullRequestRun.ts

```diff
@@ -0,0 +1,114 @@
+import { context, getOctokit } from '@actions/github';
+import { SarifRun } from 'cfn-guard';
+import getConfig from './getConfig';
+
+enum HandlePullRequestRunStrings {
+  Error = 'Tried to handle pull request result but could not find PR context.'
```

Comment:
I wonder if we should have a central place for error strings. Seems this pattern is used more .

Reply:
Yeah I can make a shared strings or consts